trillian/migrillian/election/etcd/election.go

```diff
@@ -53,8 +53,12 @@ type Election struct {
 // the passed in context is canceled before mastership is captured.
 func (e *Election) Await(ctx context.Context) (context.Context, error) {
 	// Return an error if the Election already maintains mastership context.
```

Comment:
This comment is now a bit adrift from the purpose of the code below.  How about comments something like:
```go
func (e *Election) Await(ctx context.Context) (context.Context, error) {
	if e.done != nil {
		// There was a previous monitoring goroutine, so check its completion.
		select {
		case <-e.done:
			// Previous monitoring goroutine has completed OK.
		default:
			// Previous monitoring goroutine is still running.
			return nil, errAlreadyRunning
		}
￼  	}		￼  	
```

Reply:
Done (slightly shorter version of it).